trimTrailingSlash having issue to import

### What version of Hono are you using?

4.2.2

### What runtime/platform is your app running on?

Bun

### What steps can reproduce the bug?

Document: https://hono.dev/middleware/builtin/trailing-slash
Issue: Getting the error when test the code locally.

```
import { trimTrailingSlash } from 'hono/trailing-slash'
app.use(trimTrailingSlash())
```
After run `bun run dev`, the following error shows up:

```
[1] /Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:859
[1]     return new TSError(diagnosticText, diagnosticCodes, diagnostics);
[1]            ^
[1] TSError: ⨯ Unable to compile TypeScript:
[1] index.ts(13,35): error TS2307: Cannot find module 'hono/trailing-slash' or its corresponding type declarations.
[1] 
[1]     at createTSError (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:859:12)
[1]     at reportTSError (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:863:19)
[1]     at getOutput (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:1077:36)
[1]     at Object.compile (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:1433:41)
[1]     at Module.m._compile (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:1617:30)
[1]     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
[1]     at Object.require.extensions.<computed> [as .ts] (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:1621:12)
[1]     at Module.load (node:internal/modules/cjs/loader:1207:32)
[1]     at Function.Module._load (node:internal/modules/cjs/loader:1023:12)
[1]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12) {
[1]   diagnosticCodes: [ 2307 ]
[1] }
```



### What is the expected behavior?

_No response_

### What do you see instead?

_No response_

### Additional information

_No response_

--- a/package.json
+++ b/package.json
@@ -382,6 +382,9 @@
       "etag": [
         "./dist/types/middleware/etag"
       ],
+      "trailing-slash": [
+        "./dist/types/middleware/trailing-slash"
+      ],
       "html": [
         "./dist/types/helper/html"
       ],

